mempool/src/pool/tests/utils.rs

```diff
@@ -77,3 +77,66 @@ pub fn estimate_tx_size(num_inputs: usize, num_outputs: usize) -> usize {
     );
     result
 }
+
+// TODO this calculation is already done in ChainState, reuse it
```

Comment:
Actually the calculation comes straight from the transaction verifier, which is available now.